language-syntax.adoc

```diff
@@ -1074,6 +1074,7 @@ NXSL supports C-style pre- and post-increment and decrement operators.
 |===
 | Example| Name| Result
 |`..`| Concatenation operator| Returns the concatenation of its right and left arguments.
+|`...`| Spread operator| Used when all elements from an object or array need to be included in a new array or object or should be applied one-by-one in a function call's arguments list.
```

Comment:
It's not string operator. It's Array operator so it should be in Array chapter. And it's better to include examples from issue tracker.